src/msg/async/ProtocolV2.cc

```diff
@@ -1041,7 +1056,7 @@ bool ProtocolV2::is_queued() {
 void ProtocolV2::sign_payload(bufferlist &payload) {
   ldout(cct, 21) << __func__ << " len=" << payload.length() << dendl;
 
-  if (sign_frames() && session_security) {
+  if (false && session_security) {
```

Comment:
is this something you forgot to change or will be removed in the future?

Reply:
Yeah, this is being dropped in https://github.com/ceph/ceph/pull/26169.